src/Dofs/DofHandler.jl

```diff
@@ -59,7 +59,7 @@ end
 function field_offset(dh::DofHandler, field_name::Symbol)
     offset = 0
     for i in 1:find_field(dh, field_name)-1
-        offset += getnbasefunctions(dh.field_interpolations[i]) * dh.field_dims[i]
+        offset += getnbasefunctions(dh.field_interpolations[i])::Int * dh.field_dims[i]
```

Comment:
Even if `dh.field_interpolations[i]` here I find this weird -- all `getnbasefunctions` return `Int`... Would it be better to add a fallback `getnbasefunctions` that errors?